docker/Dockerfile

```
@@ -54,7 +54,7 @@ RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH=
 
 # Install openssl for FIPS support
 #@follow_tag(registry.redhat.io/ubi9/ubi-minimal:latest)
-FROM registry.access.redhat.com/ubi9-minimal:9.4-1194@sha256:104cf11d890aeb7dd5728b7d7732e175a0e4018f1bb00d2faebcc8f6bf29bd52 AS runtime
+FROM registry.access.redhat.com/ubi9-minimal:9.4-1227@sha256:104cf11d890aeb7dd5728b7d7732e175a0e4018f1bb00d2faebcc8f6bf29bd52 AS runtime
```

```suggestion
FROM registry.access.redhat.com/ubi9-minimal:9.4-1227@sha256:35a12657ce1bcb2b7667f4e6e0147186c1e0172cc43ece5452ab85afd6532791 AS runtime
```
I can see a newer digest. Could we use that instead?
https://catalog.redhat.com/software/containers/ubi9-minimal/61832888c0d15aff4912fe0d?container-tabs=gti